tests/test_genres.py

```diff
@@ -5,34 +5,43 @@
  get_genres_by_sub_genres, GenreList, Genre, get_secondary_genres_json
 )
 
+try:
```

Comment:
You need to DRY up this part for the test. Create a BaseTestCase class and add this to the setup function. Then All the test classes can inherit from the BaseTestCase

Reply:
Very good idea... see my last commit

Reply:
Nice work